llvm/docs/LangRef.rst

```diff
@@ -12398,6 +12399,9 @@ If the operands are integer vectors, then they are compared element by
 element. The result is an ``i1`` vector with the same number of elements
 as the values being compared. Otherwise, the result is an ``i1``.
 
+If the ``samesign`` keyword is present and the operands are not of the
+same sign then the result is a :ref:`poison value <poisonvalues>`.
+
```

Comment:
Should we note here we only infer `samesign` on unsigned predicates?

Reply:
I don't think that this belongs in LangRef.